media-libs/libcamera/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
There's no maintainer listed here, which `pkgcheck scan --commits` should warn about. See also https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602.